site-config/lungmap/dev/index/projectsEntityConfig.ts

```diff
@@ -48,5 +67,197 @@ export const projectsEntityConfig: EntityConfig = {
+        id: HCA_DCP_CATEGORY_KEY.WORKFLOW,
+        width: { max: "1fr", min: "146px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.NTagCell,
+          viewBuilder: V.buildAggregatedSpecimenDisease,
+        } as ComponentConfig<typeof C.NTagCell, ProjectsResponse>,
+        header: "Disease (Specimen)",
+        id: HCA_DCP_CATEGORY_KEY.SPECIMEN_DISEASE,
+        width: { max: "1fr", min: "146px" },
+      },
+      {
+        componentConfig: {
+          component: C.NTagCell,
+          viewBuilder: V.buildAggregatedDonorDisease,
+        } as ComponentConfig<typeof C.NTagCell, ProjectsResponse>,
+        header: "Disease (Donor)",
+        id: HCA_DCP_CATEGORY_KEY.DONOR_DISEASE,
+        width: { max: "1fr", min: "128px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.NTagCell,
+          viewBuilder: V.buildAggregatedDonorDevelopmentStage,
+        } as ComponentConfig<typeof C.NTagCell, ProjectsResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.DEVELOPMENT_STAGE,
+        id: HCA_DCP_CATEGORY_KEY.DEVELOPMENT_STAGE,
+        width: { max: "1fr", min: "148px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.BasicCell,
+          viewBuilder: V.buildAggregatedDonorCount,
+        } as ComponentConfig<typeof C.BasicCell, ProjectsResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.DONOR_COUNT,
+        id: HCA_DCP_CATEGORY_KEY.DONOR_COUNT,
+        width: { max: "1fr", min: "124px" },
+      },
+      {
+        componentConfig: {
+          component: C.BasicCell,
+          viewBuilder: V.buildEstimateCellCount,
+        } as ComponentConfig<typeof C.BasicCell, ProjectsResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.EFFECTIVE_CELL_COUNT,
+        id: HCA_DCP_CATEGORY_KEY.EFFECTIVE_CELL_COUNT,
+        width: { max: "1fr", min: "124px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.BasicCell,
+          viewBuilder: V.buildAggregateSubmissionDate,
+        } as ComponentConfig<typeof C.BasicCell, ProjectsResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.AGGREGATE_SUBMISSION_DATE,
+        id: HCA_DCP_CATEGORY_KEY.AGGREGATE_SUBMISSION_DATE,
+        width: { max: "1fr", min: "224px" },
+      },
+      {
+        columnVisible: true,
+        componentConfig: {
+          component: C.BasicCell,
+          viewBuilder: V.buildAggregateLastModifiedDate,
+        } as ComponentConfig<typeof C.BasicCell, ProjectsResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.AGGREGATE_LAST_MODIFIED_DATE,
+        id: HCA_DCP_CATEGORY_KEY.AGGREGATE_LAST_MODIFIED_DATE,
+        width: { max: "1fr", min: "224px" },
+      },
+    ],
+    defaultSort: {
+      desc: SORT_DIRECTION.DESCENDING,
+      id: HCA_DCP_CATEGORY_KEY.AGGREGATE_LAST_MODIFIED_DATE,
+    },
+  },
+  listView: {
+    subTitleHero: hcaSubTitleHero,
+  },
```

Comment:
Hi @jpaten would you mind removing `listView` from the `projectsEntityConfig`; lines `258-260`.